style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css?family=Lobster%7CManjari&display=swap');
+
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  background: #ffe1ec;
+  font-family: 'Manjari', sans-serif;
+  font-size: 1.6rem;
+  color: #ffffff;
+}
+
+div {
+  margin: 0;
+  padding: 0;
+}
+
+h1, h2, h3, h4 ,h5 {
+  text-align: center;
+  font-family: 'Lobster', serif;
+  color: #ffffff;
+}
+
+h1 {
+  margin: 0 auto 30px auto;
+  font-size: 10rem;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 30px 0 50px 0;
+  width: 80%;
+}
+
+.header {
+  background: #ffa7d5;
+}
+
+form {
+  padding: 30px 0 40px 0;
+}
+
+form label, form select, form input {
+  display: block;
+  margin: 0 auto;
+}
+
+form label {
+  padding-bottom: 10px;
+  text-align: center;
+  font-size: 2.2rem;
+}
+
+.selector select {
+  margin: 0 auto;
+  height: 30px;
+  width: 250px;
+  background: #ffe1ec;
+  font-size: 2rem;
+  color: #761f46;
+  border: none;
+  border-radius: 10px;
```

Comment:
this line isn't doing anything (in my browser, at least)

Reply:
Seems to be needed in firefox